modules/interop-specs2/src/weaver/interop/specs2/Matchers.scala

```diff
@@ -0,0 +1,28 @@
+package weaver.interop.specs2
+
+import cats.Monoid
+import cats.data.Validated
+import cats.effect.IO
+import weaver.EffectSuite
+import org.specs2.matcher.MustMatchers
+import org.specs2.matcher.MatchResult
+import weaver.AssertionException
+import weaver.Expectations
+import weaver.SourceLocation
+
+trait Matchers[F[_]] extends MustMatchers {
```

Comment:
that's pretty cool, I had no idea `specs2` was serving its matchers in an independent artifact, nice find :+1: